pkg/cmd/cli/cmd/process.go

```diff
@@ -58,5 +100,34 @@ Examples:
 		},
 	}
 	cmd.Flags().StringP("filename", "f", "", "Filename or URL to file to use to update the resource")
+	cmd.Flags().StringP("value", "v", "", "Set value for the specified parameter")
+	cmd.Flags().BoolP("print", "p", false, "Do not process but only print available parameters")
```

Comment:
@smarterclayton the point is to print just the parameters, so users know what parameters they can set/override with "-v"